### PR TITLE
Fix for queries with ascending/descending order specifed

### PR DESCRIPTION
**Pull request by raufdean**
Added array sort function to honour queries which have ascending/descending specified.

### CODE REVIEW

File: src/parse-mockdb.js [hunk 1]
```diff
@@ -559,6 +559,34 @@ function queryMatchesAfterIncluding(matches, includeClause) {
...
+    let sortVal = keyVal;
+
+    // Attempt to handle dates, numbers and strings
+    const keyDate = new Date(keyVal);
+    if (keyDate !== 'Invalid Date' && !isNaN(keyDate)) {
+      sortVal = keyDate.getTime();
+    }
+    sortArray.push({ dataItem: item, sortVal });
+  });
+  sortArray.sort((a, b) => (a.sortVal - b.sortVal) * sortDir);
```
Comment (TylerBrock) [inline src/parse-mockdb.js]: Does this only work for numbers? Surely "a" - "b" does not work.

Comment (raufdean) [inline src/parse-mockdb.js]: Yes, I'll take another look at this when I have a minute and update the code and tests to properly handle strings...

Comment (TylerBrock) [inline src/parse-mockdb.js]: Great, thanks!

**Comment by TylerBrock** [comment]
Seems like a good improvement. Thanks!

**Comment by raufdean** [comment]
Thanks Tyler,

Great library - Proving very useful!

Rauf

**Pull request merged**